server/src/test/java/org/eclipse/lsp/cobol/usecases/TestIdmsCopy.java

```diff
@@ -43,7 +43,6 @@
           + "             COPY IDMS SUBSCHEMA-RECORDS\n"
           + "             COPY IDMS SUBSCHEMA-LR-CTRL SIZE 256\n"
           + "             COPY IDMS TRANSACTION-STATISTICS.\n"
-          + "             03 COPY IDMS LR ORDITEM REDEFINES OIREC\n"
```

Comment:
Why?

Reply:
There are no *OIREC* variable in this file.

Reply:
The presumption is that when the COPY IDMS statement expands, it would have the record element that is redefined.

Reply:
So does it mean that COPY IDMS implicitly brings some variable definitions?

Reply:
Yes, when the DMLC precompiler expands the COPY IDMS statement prior to the cobol compile, it can bring in variable definitions (much like a copybook).

Reply:
Could you, please, send me the list of variables that should be defined so we can add them to the predefined list and avoid false-positive errors?

Reply:
Added an error as we agreed on the meeting.